### PR TITLE
Accessibility - Internal Upload Obligations

### PR DESCRIPTION
**Pull request by NateHogan**
Should fix the following
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/45049
https://dev.azure.com/civica-digital/EA%20-%20WEEE/_workitems/edit/45047

### CODE REVIEW

File: src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml [hunk 1]
```diff
@@ -6,47 +6,50 @@
...
-    </p>
+    @using (Html.BeginForm("UploadObligations", "Obligations", FormMethod.Post))
+    {
+        @Html.AntiForgeryToken()
+        @Html.HiddenFor(m => m.Authority)
+        <p>
+            To upload your PCS's obligations, you need to first download the CSV template for the current compliance year. After this you can fill in the template with the data and upload it.
+        </p>
+        <p>
+            <button aria-hidden="true" class="link-submit govuk-link govuk-link--no-visited-state" type="submit">Download template</button>
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:27]: do you need to add tabIndex = -1?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml (hunk 1 above)
Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:27]: do you need to add this class too class="govuk-visually-hidden" ?

Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:27]: Probably remove the aria-hidden attribute as that hides it for screen readers.

Comment (meglenapnovoselska) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml:27]: ignore my comments then

Comment (NateHogan) [reply src/EA.Weee.Web/Areas/Admin/Views/Obligations/UploadObligations.cshtml]: removed attribute

**Pull request merged**